rpcserver/rpcserver.go

```diff
@@ -38,6 +38,9 @@ func (h *RPCServerHandler) InsertHeaders(
 		} else {
 			log.Info().Msgf("Inserted block header %s", blockHeader.BlockHash())
 		}
+
+		// update last checkpointed block height
```

Comment:
either this comment is missleading or the name of the function below is wrong

Reply:
`CleanUpFork()` does both, cleans up forks and updates the last checkpointed height in process. Here last checkpointed height is revelent, so added the comment.

I can modify this to 
```suggestion
		// fork cleanup updates last checkpointed height
```

Reply:
yeah - that is better. We can also think about renaming the function.